product_docs/docs/postgres_for_kubernetes/1/installation_upgrade.mdx

```diff
@@ -14,21 +14,73 @@ originalFilePath: 'src/installation_upgrade.md'
 
 ## Installation on Kubernetes
 
+### Obtaining an EDB subscription token
+
+Installing EDB Postgres for Kubernetes requires an EDB Repos 2.0 token to gain access to the EDB private software repositories. 
+You can obtain the token by visiting the [EDB Repos website](https://www.enterprisedb.com/repos) and signing into your account.
+Your account profile page displays the token to use - by default it is obscured, click the "Show" button (an eye icon) to reveal it.
+
+Your token entitles you to access one of two repositories: standard or enterprise. 
+
+* Standard - Includes the operator and operands for EDB Postgres Extended.
+* Enterprise - Includes the operator and operands for EDB Postgres Advanced and EDB Postgres Extended.
```

Comment:
both include the standard operands, as well as the operators. Worth mentioning?

Reply:
Applied a tweak to the text.